### ISSUE TITLE
so easy to broke db?

### ISSUE THREAD

**Issue by futureglorydev** [orientechnologies/orientdb#8080]
### OS: <os here>  
debian 9, orietdb 3.0
## Expected behavior  
working condition
## Actual behavior  

Once I decided to alter column type.
I dont remember exactly the types but it could be byte and integer.
Then somehow I got error about value range and after page reload I saw that orientdb saved this range..
I was working with db for a days without critical errors except that I always saw the same errors after altering column.  And recently I decided to reload db server... Since I cant connect to the db from anywhere.
I see this error and cannot access to the db  and I dont even know where error is.
the error:
```
code:500
content:"java.lang.NumberFormatException: Value out of range. Value:"256" Radix:10"
reason:500
```
lost my DB
## Steps to reproduce  
<add here>

**Comment by laa** [comment]
@futureglorydev could you provide me exception which you get now? Do you work on a single server or distributed database?

**Comment by laa** [comment]
@futureglorydev could you send me server log? Are you able to access DB from `plocal:` connection?

**Comment by futureglorydev** [comment]
Its single server.
Exact version of DB:

> OrientDB console v.3.0.0m1 (build 63b896d280caa08f511517a6e35978d214ffea47)


Error from the console (after I got error about range):

> orientdb> connect plocal://opt/orientdb/databases/my_db root root_password
> 
> 
> Connecting to database [plocal://opt/orientdb/databases/my_db] with user 'root'...
> Error: com.orientechnologies.orient.core.exception.ODatabaseException: Cannot open database 'my_db'
> 
> Error: com.orientechnologies.orient.core.exception.OStorageException: Database is locked by another process, please shutdown process and try again

Error from the console (first time error, after rebooting db):

> orientdb> connect plocal://opt/orientdb/databases/my_db root root_password
> 
> 
> Connecting to database [plocal://opt/orientdb/databases/my_db] with user 'root'...
> Error: com.orientechnologies.orient.core.exception.ODatabaseException: Cannot open database 'my_db'
> 
> Error: com.orientechnologies.orient.core.exception.ODatabaseException: Cannot open database url=plocal:/opt/orientdb/databases/my_db
> 
> Error: java.lang.NumberFormatException: Value out of range. Value:"256" Radix:10

**Comment by futureglorydev** [comment]
@laa 
full log
[log.zip](https://github.com/orientechnologies/orientdb/files/1703542/log.zip)

**Comment by futureglorydev** [comment]
@laa 
I`ve got the same error on new RC1 version on my new just created DB
I just added new datetime attribute to my class and I got an error
http://joxi.ru/Y2Lz9a9tnPXJBr
After restarting I cant access my db again..

**Comment by laa** [comment]
@futureglorydev in your case there are three issues. First of all RC1 version is quite old, all fixes are in the latest snapshot (we pushed changes a few days ago). We are going to release RC2 soon. Second I suppose that you crashed DB (according to the log) and it was restored incorrectly. And third I suppose you have issues with database configuration. I will ping you when we release RC2 so you can retry. Could you send me your DB btw so I will inspect it?

**Comment by laa** [comment]
@futureglorydev I have been working on durability feature last days, so your issue appeared just in time, but if your database is not restored again after DB crash, probably that is because you use relatively old version. But your issue with "Value out of range" is something different. I would be glad if you send me the database to inspect it.

**Comment by futureglorydev** [comment]
@laa I would love to restore but I cant do it.
Here is 2 databases  both are broken
[sk_dev.zip](https://github.com/orientechnologies/orientdb/files/1719972/sk_dev.zip)
[test.zip](https://github.com/orientechnologies/orientdb/files/1719973/test.zip)

**Comment by futureglorydev** [comment]
@laa I think it happened after I added attribute with type of 'BYTE' and pointed range 0-256(I thought its possible but max value is 127 even though value range starts from 0). After deletion this filed db goes to normal. Is it possible to restore DB without accessing to it?

**Comment by laa** [comment]
Fixed in 3.0
